components/ItemMenu.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <v-menu absolute>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn
```

Comment:
This is causing an ESLint error. However changing it to `#:activator` breaks it :P